Makefile

```diff
@@ -71,7 +75,34 @@ RELEASE_CMD=curl -sL https://git.io/goreleaser | GOPATH=$(mktemp -d) VERSION=$(G
 publish:
 	$(RELEASE_CMD)
 
+GIT_COMMAND=git
```

Comment:
This looks good and it looks like it should work. However, it does bloat the Makefile, so I have two questions:
1. Should this live in a separate shell script?
2. If it makes it easier and more maintainable in the future, is it possible we implement this in node js?

What do you think?

Reply:
As discussed, let's move this to a separate script to keep the core Makefile small and go ahead with the merge. Thanks @InTheCloudDan !